Jenkinsfile

```diff
@@ -13,7 +13,7 @@ buildMvn {
     buildDocker {
       publishMaster = 'yes'
       healthChk = 'no'
-      healthChkCmd = 'curl -sS --fail -o /dev/null  http://localhost:8081/apidocs/ || exit 1'
+      healthChkCmd = 'wget --no-verbose --tries=1 --spider http://localhost:8081/admin/health || exit 1'
```

Comment:
is changing "apidocs/" to "admin/health" intended?

Reply:
fisrt I little bit confuse, then I saw there in this https://issues.folio.org/browse/FOLIO-3407 place and I also changed it

Reply:
please make sure this url is available in the module (looks like you won't see the error during build anyway since healthcheck is disabled: healthChk = 'no')

Reply:
Ok, looks like these url avilable 
![image](https://user-images.githubusercontent.com/113523904/214013070-6d9f8e58-e764-4234-8726-b9afeab60ac2.png)